src/main/java/de/bonndan/nivio/output/map/PNGRenderCache.java

```diff
@@ -71,16 +69,15 @@ public PNGRenderCache(MapFactory<mxGraph, mxCell> mapFactory, MapStyleSheetFacto
      * @return the svg as string, uncached
      */
     public String getSVG(LandscapeImpl landscape) {
-        JGraphXRenderer jGraphXRenderer = new JGraphXRenderer();
-        RenderedArtifact<mxGraph, mxCell> render = jGraphXRenderer.render(landscape);
-        mapFactory.applyArtifactValues(landscape, render);
+        OrganicLayouter layouter = new OrganicLayouter();
```

Comment:
Why don't we use the interface `Layouter`?

Reply:
I've pushed a change. Please see if you agree with it. This code is likely to change later when there are more than just one layouter and render implementations.

Reply:
Yes, if instantiation in the constructor makes sense because we don't need a fresh layouter each time we call `getSVG()`, sure, why not. This was just intended as a question to understand how we are using interfaces, and so on :)